iothub/service/src/Common/Security/ISharedAccessSignatureCredential.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Azure.Devices.Common.Security
 
     using Microsoft.Azure.Devices.Common.Data;
       
-    public interface ISharedAccessSignatureCredential
+    internal interface ISharedAccessSignatureCredential
```

Comment:
Is this change intentional?

Reply:
yes. If we want to make SharedAccessSignatureAuthorizationRule internal, we have to make this internal. And it seems like that visibility is sufficient.

Reply:
Well, we shouldn't be making any types that were public not public any more, unless we have reason to believe the types were of no use.